src/status_im/contexts/profile/login/events.cljs

```diff
@@ -45,12 +45,10 @@
          log-level (or (:log-level settings) config/log-level)
          pairing-completed? (= (get-in db [:syncing :pairing-status]) :completed)]
      {:db (cond-> (-> db
-                      (assoc :chats/loading?           true
-                             :networks/current-network current-network
-                             :networks/networks        (merge networks config/default-networks-by-id)
-                             :profile/profile          (merge profile-overview
-                                                              settings
-                                                              {:log-level log-level}))
+                      (assoc :chats/loading?  true
+                             :profile/profile (merge profile-overview
+                                                     settings
+                                                     {:log-level log-level}))
```

Comment:
These network keys not required anymore?

Reply:
We shouldn't need them afaik. We put all the related network pieces into :wallet/networks as far as I know. There might be some code touching off these that I will double check, but realistically we should not be using any legacy subs so it's better we move away from this 👍

Reply:
Yeah, `:networks/networks` contains the L1 chains data (used in the old wallet) which is outdated now.